torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -198,6 +202,102 @@ bool is_numpy_scalar(PyObject* obj) {
           PyArray_IsScalar(obj, Floating));
 }
 
+at::Tensor tensor_from_cuda_array_interface(PyObject* obj) {
+  PyObject *cuda_dict = PyObject_GetAttrString(obj, "__cuda_array_interface__");
```

Comment:
You have an owning reference and must free it on all exit paths (the easiest way to make sure this happens is to use one of pybind11's RAII owning reference types!)

Reply:
Fixed in 2cf1117